sources/packages/backend/libs/sims-db/src/entities/student-appeal.model.ts

```diff
@@ -42,7 +42,7 @@ export class StudentAppeal extends RecordDataModel {
    */
   @ManyToOne(() => Application, {
     eager: false,
-    cascade: false,
+    cascade: ["update"],
```

Comment:
What is the portion of code which is going to use the benefit of this cascade update?

Reply:
Updates the `currentAssessment` on the application table.
```ts
appealToUpdate.application.currentAssessment = newAssessment`
```
https://github.com/bcgov/SIMS/blob/feature/%231683-application-complete-tracker/sources/packages/backend/apps/api/src/services/student-appeal/student-appeal.service.ts#L396